resources/services/storage/storage_metrics.go

```diff
@@ -0,0 +1,157 @@
+package storage
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cast"
+
+	"github.com/cloudquery/cq-provider-gcp/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+	"google.golang.org/api/monitoring/v3"
+)
+
+const (
+	queryACLCount = `
+		fetch gcs_bucket
+		| metric 'storage.googleapis.com/authz/acl_operations_count'
```

Comment:
So I think the query you are using will also find instances of bucket acls granting access. I think we want `acl_based_object_access_count` where it is
```
Delta count of requests that result in an object being granted access solely due to object ACLs
```

Reply:
Updated to acl_based_object_access_count  metric